src/prediction_service_utils.cpp

```diff
@@ -72,13 +72,21 @@ Status getModelInstance(ovms::ModelManager& manager,
         if (modelInstance == nullptr) {
             return StatusCode::MODEL_VERSION_MISSING;
         }
-    } else {
+
+        return modelInstance->waitForLoaded(WAIT_FOR_MODEL_LOADED_TIMEOUT_MS, modelInstanceUnloadGuardPtr);
+    } 
+    
+    Status status = StatusCode::MODEL_VERSION_NOT_LOADED_ANYMORE;
+    auto retries = 5;
```

Comment:
Perhaps we could have this 5 in a constant variable? Or load it from environment variable? What do you think?

Reply:
added a const value.